sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.bulkInsert.e2e-spec.ts

```diff
@@ -317,6 +321,101 @@ describe("EducationProgramOfferingInstitutionsController(e2e)-bulkInsert", () =>
+        "bulk-insert/single-upload-with-max-exceeding-offering-cost.csv",
+      );
+      // Act/Assert
+      await request(app.getHttpServer())
+        .post(`${endpoint}?validation-only=true`)
+        .attach("file", singleOfferingWithMaxExceedingOfferingCost)
+        .auth(institutionUserToken, BEARER_AUTH_TYPE)
+        .expect(HttpStatus.UNPROCESSABLE_ENTITY)
+        .expect({
+          message: "An offering has invalid data.",
+          errorType: OFFERING_VALIDATION_CRITICAL_ERROR,
+          objectInfo: [
+            {
+              recordIndex: 0,
+              locationCode: csvLocationCodeYESK,
+              sabcProgramCode: csvProgramSABCCodeSBC1,
+              startDate: "2023-09-02",
+              endDate: "2024-08-12",
+              offeringStatus: OfferingStatus.CreationPending,
+              errors: [],
+              infos: [],
+              warnings: [
+                {
+                  typeCode:
+                    OfferingValidationWarnings.OfferingCostExceedMaximum,
+                  message:
+                    "programRelatedCosts must not be greater than 100000",
+                },
+                {
+                  typeCode:
+                    OfferingValidationWarnings.ProgramOfferingDeliveryMismatch,
+                  message:
+                    "Delivery type has an offering delivery that is not allowed by its program.",
+                },
+              ],
+            },
+          ],
+        });
+    },
+  );
+
+  it(
+    "Should return validation error when bulk offering CSV file with existing location code and SABC " +
+      `code with mandatory fees greater than ${MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE} and different delivery method is uploaded.`,
+    async () => {
+      // Arrange
+      const singleOfferingErrorWhenOfferingCostExceedMax = path.join(
+        __dirname,
+        "bulk-insert/single-upload-error-when-offering-cost-exceed-max.csv",
+      );
+      // Act/Assert
+      await request(app.getHttpServer())
+        .post(`${endpoint}?validation-only=true`)
+        .attach("file", singleOfferingErrorWhenOfferingCostExceedMax)
+        .auth(institutionUserToken, BEARER_AUTH_TYPE)
+        .expect(HttpStatus.UNPROCESSABLE_ENTITY)
+        .expect({
+          message: "An offering has invalid data.",
+          errorType: OFFERING_VALIDATION_CRITICAL_ERROR,
+          objectInfo: [
+            {
+              recordIndex: 0,
+              locationCode: csvLocationCodeYESK,
+              sabcProgramCode: csvProgramSABCCodeSBC1,
+              startDate: "2023-09-02",
+              endDate: "2024-08-12",
+              errors: ["Mandatory fees must be not greater than 999999."],
+              infos: [],
+              warnings: [
+                {
+                  typeCode:
+                    OfferingValidationWarnings.OfferingCostExceedMaximum,
+                  message: "mandatoryFees must not be greater than 100000",
+                },
```

Comment:
As mentioned on https://github.com/bcgov/SIMS/pull/2529/files#r1404683716 I think this is weird for the person who is checking those messages.

Reply:
ok. Thanks for the explanation.